src/main/java/bio/terra/cli/service/GoogleBqDatasets.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.cli.service;
+
+import bio.terra.cli.exception.SystemException;
+import bio.terra.cli.service.utils.CrlUtils;
+import bio.terra.cloudres.google.bigquery.BigQueryCow;
+import com.google.api.services.bigquery.model.TableList;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.bigquery.DatasetId;
+import java.io.IOException;
+
+public class GoogleBqDatasets {
```

Comment:
nit: public class should have a description. Also perhaps rename this class to `GoogleBigQuery` since that's the service name.